lib/matplotlib/axes/_axes.py

```diff
@@ -1883,96 +1883,83 @@ def bar(self, *args, **kwargs):
 
            (x - width/2, x + width/2, bottom, bottom + height)
 
-        (left, right, bottom and top edges) by default.  *x*,
-        *height*, *width*, and *bottom* can be either scalars or
-        sequences.
+        (left, right, bottom and top edges) by default.  *x*, *height*,
```

Comment:
would also remove the `math` markup just before, whose html rendering is a bit unwieldy, and replace it by e.g. inline double-backquotes.

Reply:
The sentence is a bit technical anyway. Nobody thinks of a bar plot in terms of rectangle coordinates. Will switch to a more semantic description.